src/components/atoms/Timeline/index.tsx

```diff
@@ -1,8 +1,11 @@
 import { memo } from "react";
 
-import { styled } from "@reearth/theme";
+import { useT } from "@reearth/i18n";
+import { PublishTheme, styled, usePublishTheme } from "@reearth/theme";
 
+import { SceneProperty } from "../../molecules/Visualizer";
```

Comment:
Please use `@reearth/components` for URLs that aren't heavily nested AND tightly related. (example: its fine that ScaleList.tsx is imported relatively)
While you are at it please update the other atoms' imports as well.

Reply:
This is fixed by [chore: update some import format](https://github.com/reearth/reearth-web/pull/277/commits/fa94d7008f59a430ecdc9db46bf2a464d98a1b0b)